### ISSUE TITLE
VET accessibility - Images used purely for decoration should have null alt text

### ISSUE THREAD

**Issue by mhk33** [cul-it/uls#47]
This was open in JIRA, not sure if it was fixed.

https://culibrary.atlassian.net/browse/ULS-57

**Comment by manolobevia** [comment]
CULU theme will take care of this after migrating vet site to the new CULU theme.
